src/Yoti/Entity/Anchor.php

```diff
@@ -4,33 +4,43 @@
 
 class Anchor
 {
+    const TYPE_SOURCE_OID = '1.3.6.1.4.1.47127.1.1.1';
```

Comment:
We shouldn't need these two constants to be defined in two places

Reply:
Which two places?

Reply:
Anchor and AnchorProcessor

Reply:
There is no ore AnchorProcessor - it's been deleted !